src/client/components/componentWithSliders/modalsavewindow/modalsavewindow.css

```diff
@@ -0,0 +1,84 @@
+.modalSaveWindow {
```

Comment:
Actually, I saw what we all using naming of classes/id in components and CSS styles with the big first letter.
In my opinion, we have to use the same style, like: 
.Upholder >.Upholder, #modesItem > #ModesItem and etc.

Reply:
not all, only a few, look to the components.